[2025-03-20T19:30:46.421-0500] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2025-03-20T19:30:46.431-0500] {taskinstance.py:2614} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: proceso_etl.tarea_limpieza manual__2025-03-21T00:28:19.235584+00:00 [queued]>
[2025-03-20T19:30:46.437-0500] {taskinstance.py:2614} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: proceso_etl.tarea_limpieza manual__2025-03-21T00:28:19.235584+00:00 [queued]>
[2025-03-20T19:30:46.438-0500] {taskinstance.py:2867} INFO - Starting attempt 1 of 2
[2025-03-20T19:30:46.461-0500] {taskinstance.py:2890} INFO - Executing <Task(PythonOperator): tarea_limpieza> on 2025-03-21 00:28:19.235584+00:00
[2025-03-20T19:30:46.470-0500] {standard_task_runner.py:72} INFO - Started process 52141 to run task
[2025-03-20T19:30:46.473-0500] {standard_task_runner.py:104} INFO - Running: ['airflow', 'tasks', 'run', 'proceso_etl', 'tarea_limpieza', 'manual__2025-03-21T00:28:19.235584+00:00', '--job-id', '51', '--raw', '--subdir', 'DAGS_FOLDER/dag_etl.py', '--cfg-path', '/tmp/tmp7wsziq_x']
[2025-03-20T19:30:46.475-0500] {standard_task_runner.py:105} INFO - Job 51: Subtask tarea_limpieza
[2025-03-20T19:30:46.512-0500] {task_command.py:467} INFO - Running <TaskInstance: proceso_etl.tarea_limpieza manual__2025-03-21T00:28:19.235584+00:00 [running]> on host luis-GF65
[2025-03-20T19:30:46.581-0500] {taskinstance.py:3134} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='airflow' AIRFLOW_CTX_DAG_ID='proceso_etl' AIRFLOW_CTX_TASK_ID='tarea_limpieza' AIRFLOW_CTX_EXECUTION_DATE='2025-03-21T00:28:19.235584+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2025-03-21T00:28:19.235584+00:00'
[2025-03-20T19:30:46.582-0500] {taskinstance.py:732} INFO - ::endgroup::
[2025-03-20T19:30:49.062-0500] {limpieza.py:53} INFO - 1. Terminando pre-limpieza antes del análisis !!
[2025-03-20T19:30:49.113-0500] {limpieza.py:74} INFO - 2. Terminando corrección de coordenadas y distrito !!
[2025-03-20T19:30:49.122-0500] {limpieza.py:92} INFO - Relación en distrito barranco: 2.29
[2025-03-20T19:30:49.122-0500] {limpieza.py:96} INFO - Relación global entre preciosxm2: 2.29
[2025-03-20T19:30:49.124-0500] {limpieza.py:103} INFO - 3. Terminando corrección de precios extremadamente bajos !!
[2025-03-20T19:30:49.135-0500] {logging_mixin.py:190} WARNING - /home/luis/Proyectos/PrecioDeDepartamentos/Precios-depas-git/ProyectoDepas/2.0.StaggingLayer/StagingLayerProject/Pipeline/tasks/limpieza.py:116 SettingWithCopyWarning: 
A value is trying to be set on a copy of a slice from a DataFrame.
Try using .loc[row_indexer,col_indexer] = value instead

See the caveats in the documentation: https://pandas.pydata.org/pandas-docs/stable/user_guide/indexing.html#returning-a-view-versus-a-copy
[2025-03-20T19:30:49.146-0500] {limpieza.py:130} INFO - 4. Terminando tratamiento de duplicados !!
[2025-03-20T19:30:49.146-0500] {logging_mixin.py:190} INFO - No existen valores nulos en area
[2025-03-20T19:30:49.147-0500] {logging_mixin.py:190} INFO - No existen valores nulos en antiguedad
[2025-03-20T19:30:49.158-0500] {logging_mixin.py:190} WARNING - /home/luis/Proyectos/PrecioDeDepartamentos/Precios-depas-git/ProyectoDepas/2.0.StaggingLayer/StagingLayerProject/Pipeline/tasks/limpieza.py:196 FutureWarning: Series.__getitem__ treating keys as positions is deprecated. In a future version, integer keys will always be treated as labels (consistent with DataFrame behavior). To access a value by position, use `ser.iloc[pos]`
[2025-03-20T19:30:49.179-0500] {limpieza.py:213} INFO - 7. Terminando última limpieza antes de EDA !!
[2025-03-20T19:30:50.890-0500] {python.py:240} INFO - Done. Returned value was: None
[2025-03-20T19:30:50.894-0500] {taskinstance.py:341} INFO - ::group::Post task execution logs
[2025-03-20T19:30:50.895-0500] {taskinstance.py:353} INFO - Marking task as SUCCESS. dag_id=proceso_etl, task_id=tarea_limpieza, run_id=manual__2025-03-21T00:28:19.235584+00:00, execution_date=20250321T002819, start_date=20250321T003046, end_date=20250321T003050
[2025-03-20T19:30:50.975-0500] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2025-03-20T19:30:50.990-0500] {taskinstance.py:3901} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2025-03-20T19:30:51.008-0500] {local_task_job_runner.py:245} INFO - ::endgroup::
